### DISCUSSION TITLE
处理token过期问题

### DISCUSSION THREAD

**Discussion by woaimi520** [liangjingkanji/Net#119] [General]
处理token过期问题,如果是在请求服务端 根据返回的code值来判断 是否token过期,如果过期 就刷新token 然后重新请求,如果token刷新失败 就让重新登录 , 这块有没有实现好的demo 参考下,如果用拦截器 可以完成吗

**Comment by liangjingkanji** [comment]
业务逻辑请自行搜索OkHttp如何实现Token自动刷新, 不同业务不同实现, 网上实现方式非常多甚至可以直接复制粘贴

Net兼容OkHttp的所有实现方式和方法组件

**Comment by liangjingkanji** [comment]
补充文档: [刷新token拦截器示例](https://liangjingkanji.github.io/Net/interceptor/)
